### ISSUE TITLE
Call logs don't seem to be backed up 

### ISSUE THREAD

**Issue by GCollinson** [jberkel/sms-backup-plus#632]
Hi. Very long term, happy user of sms backup +. 
Just changed from Samsung Galaxy S5 to Samsung Galaxy S7 Edge. 
I've noticed that call logs are no longer  appearing in my Google calendar. 
Sms messages are still being backed up. 

Searching my calendar on laptop, it looks like last "missed call"  was saved to calendar on 22nd October 2015. 

(I use the feature a lot while working, but have been out of contract since August 2015 so not checked it for a while) 

It looks like last "called" entry in calendar is Monday 5th October 2015.

Last "call from" entry is Monday 11th October 2015. 

Is anyone else having this issue? 

Google calendar code change perhaps? 

It's clearly not due to my S7 upgrade, but (for me at least) a much longer issue. 

As I say, sms are still being backed up even in last day or so, despite phone upgrade being last week.

Thanks 

Glen

**Comment by GCollinson** [comment]
I've just checked my Gmail "call-log" label and it looks like all of my call history IS in there... 

So fortunately I have got backups of calls made, received and missed. 

However, they no longer appear in my calendar. 

Any idea 

1. Why it's stopped working 
2. How to get it working 
3. How to backfill the entries that are missing from the calendar, but exist in Gmail label? 

Many thanks 


Glen

**Comment by oohgodyeah** [comment]
I had this same problem on Android 4.0.4, app version 1.5.10.  Call logs stopped syncing to my Google calendar between March 28, 2016 through April 23.  But all the call logs appear in my Gmail under the Call Log tag.  It would love to be able to re-sync these to my calendar for billing purposes.

**Comment by Kolossi** [comment]
TL:DR try ensuring that Settings->Auto Backup Settings->3rd party integration is enabled.

New SMS BAckup+ user.

I was getting this issue, and was suspecting that if an sms didn't come in on a given day to trigger a backup, SMS Backup+ would "lose touch" with its calendar sync.

So I made a Tasker profile to force an SMS Backup+ backup at the end of every call.  Tried it and it didn't work.

Investigating it I found that being a performance/security optimisation kinda guy I always go through all settings in new apps and turn off anything I don't need.  This had included the setting "3rd party integration" which I had turned off.

Turned it back on and the Tasker task works fine and I get Calendar sync of call log entries.  Debugging another profile in Tasker I turned tasker off for a couple of days but found that actually SMS Backup+ was still doing calendar sync fine.

So without doing further testing (I'm working fine now thanks :-) ) it would appear that the "3rd party integration" needs to be turned on for calender sync - for geeks maybe SMS Backup+ uses "intents" as part of its internal implementation of this?

TL:DR try ensuring that Settings->Auto Backup Settings->3rd party integration is enabled

**Comment by oohgodyeah** [comment]
Thanks for the suggestion.  3rd Party Integration is not enabled for my phone and never has been, yet my logs have been syncing except for the gap mentioned previously.  My understanding of this [feature ](https://github.com/jberkel/sms-backup-plus/blob/master/README.md)from is that is not needed unless integrating with other apps: "If you want to trigger backups from another app, enable 3rd party integration in Advanced Settings and send the broadcast intent com.zegoggles.smssync.BACKUP. This will work even when Auto Backup is disabled."

At this point, the Call Logs started syncing on their own without 3rd party integration enabled.  Like OP, I just want to get the missing call logs messages in Gmail synced over to my Google Calendar.
